src/pages/posts/sixth-post.md

```diff
@@ -118,7 +118,7 @@ Some text to show that the reference links can follow later.
 Images included in _\_posts_ folder are lazy loaded.
 
 Inline-style:
-![alt text](/src/images/random.jpeg 'Logo Title Text 1')
+![alt text](http://localhost:3000/src/images/random.jpeg 'Logo Title Text 1')
```

Comment:
I think `http://localhost:3000` should be removed. Otherwise, everything else looks good to me.

Reply:
We could point to the repos url instead what you think ?

Reply:
If you juste remove the domain the page won't load and thus break the app

Reply:
@Izocel, we should just remove the domain. If we remove the domain and if the image won't load, this mean there is a bug.

Putting `localhost:3000` is not a solution because it will break in production with a real domain.